src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -192,6 +192,22 @@ export default class ReplicationControllerDetailController {
    * @export
    */
   hasMemoryUsage(pod) {
-    return !!pod.metrics && (!!pod.metrics.memoryUsage || pod.metrics.memoryUsage === 0);
+    return !!pod.metrics && !!pod.metrics.memoryUsageHistory && pod.metrics.memoryUsageHistory.length > 0;
+  }
+
+  /**
+   * Produces the last value of the last element of an array of MetricsResults.
+   * @param {!Array<!backendApi.MetricResult>} ts
+   * @return {number}
+   * @export
+   */
+  latestTimeseriesValue(ts) {
```

Comment:
How about to rely on what backend returns in the cpuUsage field in JSON. Let the backend decide what to display and frontend only care how to display it.